### PR TITLE
LPS-163891 add validation to Object Field to show error when published with empty field

### PR DESCRIPTION
**Pull request by manuelecastro**
https://issues.liferay.com/browse/LPS-163891

### CODE REVIEW

**Comment by manuelecastro** [comment]
ci:test:sf

**Comment by manuelecastro** [comment]
ci:test:sf

**Comment by manuelecastro** [comment]
ci:test:sf

**Comment by manuelecastro** [comment]
ci:test:relevant

**Comment by manuelecastro** [comment]
ci:test:relevant

**Review by danvb112 — APPROVED** [review]
LGTM!

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/hooks/useValidateFormWithObjects.js [hunk 1]
```diff
@@ -75,7 +75,10 @@ export function useValidateFormWithObjects() {
 				}
 			});
 
-			if (hasObjectDefinitionId && isStorageTypeObject) {
+			if (
+				(hasObjectDefinitionId && isStorageTypeObject) ||
+				!!objectFields[0]
```
Comment (danvb112) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/hooks/useValidateFormWithObjects.js:80]: Hey @manuelecastro, you can just use `objectFields.length` to make this condition

**Review by danvb112 — CHANGES_REQUESTED** [review]
Good work on this bug @manuelecastro, I have just one suggestion!

**Comment by manuelecastro** [comment]
ci:test:sf

**Comment by manuelecastro** [comment]
ci:test:relevant

**Review by danvb112 — APPROVED** [review]
LGTM!

**Comment by manuelecastro** [comment]
ci:forward

**Pull request closed without merging**